hclsyntax/expression_template_test.go

```diff
@@ -297,13 +297,25 @@ trim`,
 			`${greeting} ${target}`,
 			&hcl.EvalContext{
 				Variables: map[string]cty.Value{
-					"greeting": cty.StringVal("hello").WithMarks(cty.NewValueMarks("english")),
-					"target":   cty.StringVal("world").WithMarks(cty.NewValueMarks("sensitive")),
+					"greeting": cty.StringVal("hello").Mark("english"),
+					"target":   cty.StringVal("world").Mark("sensitive"),
```

Comment:
a question -- these WithMarks => Mark this merely simplify the test, not changing any behavior here yes?

Reply:
Yep, that's right!